cwl/src/main/scala/cwl/CwlExpressionCommandPart.scala

```diff
@@ -80,7 +81,7 @@ abstract class CommandLineBindingCommandPart(commandLineBinding: CommandLineBind
         case _ => prefixAsList
       }
       case _: WomObjectLike => prefixAsList
-      case _ => List.empty
+      case w => throw new RuntimeException(s"Unhandled value: $w")
```

Comment:
now explody on unhandled values rather than silently failing

Reply:
Use either `.toDisplayString` or `.womType.toDisplayString`. I'd prefer the second since this is a "you gave me the wrong type" problem